apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/TableMetadata.java

```diff
@@ -1,34 +1,31 @@
 package com.linkedin.openhouse.jobs.util;
 
-import lombok.Builder;
+import javax.annotation.Nullable;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
 
 /** Table metadata, including database name, table name and owner. */
 @Getter
+@SuperBuilder
 @EqualsAndHashCode(callSuper = true)
+@ToString
 public class TableMetadata extends Metadata {
-  String dbName;
-  String tableName;
-
-  @Builder
-  public TableMetadata(String creator, String dbName, String tableName) {
-    super(creator);
-    this.dbName = dbName;
-    this.tableName = tableName;
-  }
-
-  @Override
-  public String toString() {
-    return String.format("dbName: %s, tableName: %s, creator: %s", dbName, tableName, creator);
-  }
+  protected String dbName;
```

Comment:
Changed to make this more consistent with the way we've also designed the data layout strategy etc.?